subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r43/ToolingApiVersionSupport.groovy

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.tooling.r43
+
+import org.gradle.integtests.fixtures.ScriptExecuter
+import org.gradle.integtests.tooling.fixture.ToolingApiSpecification
+import org.gradle.integtests.tooling.r18.NullAction
+import org.gradle.tooling.ProjectConnection
+import org.gradle.tooling.model.eclipse.EclipseProject
+import org.gradle.util.GradleVersion
+
+abstract class ToolingApiVersionSupport extends ToolingApiSpecification {
+    def output = new ByteArrayOutputStream()
+
+    def setup() {
+        file("build.gradle") << noopScript()
```

Comment:
⭕️  You don't need an empty script, Gradle will work fine on any folder, even empty.

Reply:
Thanks and good to know, I just need a "task" to run in TAPI build execution.

Reply:
`help` is always available

Reply:
Aha, you're right. :-)